### ISSUE TITLE
RSpec/FactoryBot/CreateList fails with "undefined method `to_sym' for nil:NilClass"

### ISSUE THREAD

**Issue by ThomasKoppensteiner** [rubocop-hq/rubocop-rspec#1086]
The `Rspec/FactoryBot/CreateList` fails with `undefined method to_sym for nil:NilClass`.
It fails for both styles (`create_list, n_times`).

```
Mention the following information in the issue report:
1.2.0 (using Parser 2.7.2.0, rubocop-ast 1.1.1, running on jruby 2.5.7 java)
```

The following versions are used:

- JRuby `9.2.13.0`
- rubocop `1.2.0`
- rubocop-rspec `1.42.0`

**Backtrace:**
```text
org/jruby/RubyKernel.java:1009:in `load'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/bin/rubocop:23:in `<main>'
org/jruby/RubyKernel.java:1009:in `load'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `kernel_load'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:28:in `run'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/cli.rb:476:in `exec'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/cli.rb:30:in `dispatch'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/cli.rb:24:in `start'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/exe/bundle:46:in `block in <main>'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/lib/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/bundler-2.1.4/exe/bundle:34:in `<main>'
org/jruby/RubyKernel.java:1009:in `load'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/bin/bundle:23:in `<main>'

An error occurred while RSpec/FactoryBot/CreateList cop was inspecting /Users/someuser/dev/someproject/spec/support/helper.rb:12:17.
undefined method `to_sym' for nil:NilClass

/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/mixin/configurable_enforced_style.rb:69:in `style'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-rspec-1.42.0/lib/rubocop/cop/rspec/factory_bot/create_list.rb:61:in `on_send'
org/jruby/RubyKernel.java:1960:in `public_send'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:100:in `block in trigger_responding_cops'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:166:in `with_cop_error_handling'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:99:in `block in trigger_responding_cops'
org/jruby/RubyArray.java:1809:in `each'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:98:in `trigger_responding_cops'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:69:in `on_send'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-ast-1.1.1/lib/rubocop/ast/traversal.rb:158:in `block in on_send'
org/jruby/RubyArray.java:1809:in `each'
org/jruby/RubyEnumerable.java:1258:in `each_with_index'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-ast-1.1.1/lib/rubocop/ast/traversal.rb:155:in `on_send'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:71:in `on_send'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-ast-1.1.1/lib/rubocop/ast/traversal.rb:153:in `on_def'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-1.2.0/lib/rubocop/cop/commissioner.rb:71:in `on_def'
/Users/someuser/.rbenv/versions/jruby-9.2.13.0/lib/ruby/gems/shared/gems/rubocop-ast-1.1.1/lib/rubocop/ast/traversal.rb:136:in `block in on_dstr'
```

**Comment by ThomasKoppensteiner** [comment]
[Line 69](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/mixin/configurable_enforced_style.rb#L69) from the `rubocop` mixin `ConfigurableEnforcedStyle` and surroundings:

```ruby
def style
  @style ||= begin
    s = cop_config[style_parameter_name].to_sym
    raise "Unknown style #{s} selected!" unless supported_styles.include?(s)

    s
  end
end

...

def style_parameter_name
  'EnforcedStyle'
end
```

=> Looks like `cop_config[style_parameter_name]` is `nil`.

**Comment by ThomasKoppensteiner** [comment]
I think the issue is, that the `cop_config` only consists of:

```ruby
{
  "Enabled" => true
}
```

Working example:
```ruby
{
  "Description" => "Use Unix-style line endings.",
  "StyleGuide" => "#crlf",
  "Enabled" => true,
  "VersionAdded" => "0.49",
  "EnforcedStyle" => "native",
  "SupportedStyles" => ["native", "lf", "crlf"]
}"
```

**Comment by pirj** [comment]
`rubocop-rspec` 1.42 is not compatible with RuboCop 1.2. Please update to a recently released `rubocop-rspec` 2.0.

**Comment by ThomasKoppensteiner** [comment]
@pirj thank you for pointing my to the  `2.0` version. Somehow missed it.
